src/server/blockchain/AdminWallet.js

```diff
@@ -375,12 +384,9 @@ export class Wallet {
       let toTop = maxTopWei - userBalance
       log.debug('TopWallet:', { userBalance, toTop })
       if (toTop > 0 && (force || toTop / maxTopWei >= 0.75)) {
-        let res = await this.sendNative({
-          from: this.address,
-          to: address,
-          value: toTop,
-          gas: defaultGas,
-          gasPrice: defaultGasPrice
+        let res = await this.sendTransaction(this.proxyContract.methods.topWallet(address)).catch(e => {
+          log.error('Error topWallet', { e }, e.message, { address })
```

Comment:
redundant catch

Reply:
done in https://github.com/GoodDollar/GoodServer/pull/139/commits/1ad3e83aab6fc53447f5709900332c4d7a565d7c